src/partials/about&gallery.html

```diff
@@ -0,0 +1,49 @@
+<section class="how-its-made">
+        <div class="container">
+            <p class="how-its-made_label">Tradition and love</p>
+            <h2 class="how-its-made_title" id="how-its-made">How it’s made?</h2>
+            <div class="milk">
+                <div class="milk_image">
+                <picture>
+                    <source srcset="./images/mobile-about/cow_and_milk.png 1x,./images/mobile-about/cow_and_milk@2x.png 2x" media="(max-width: 767px)">
+                    <source srcset="./images/tablet-about/cow_and_milk.png 1x,./images/tablet-about/cow_and_milk@2x.png 2x" media="(min-width: 768px)">
+                    <source srcset="./images/desctop-about/cow_and_milk.png 1x,./images/desctop-about/cow_and_milk@2x.png 2x" media="(min-width: 1200px)">
+                    <img class="main-image" src="./images/mobile-about/cow_and_milk.png" alt="cow and milk">
+                </picture>
+            </div> 
+            <div class="milk_description">
+            <p class="milk_description-first">Fusce ut velit laoreet, tempus arcu eu, molestie tortor. Nam vel justo cursus, faucibus lorem
+                eget, egestas eros.
+                Maecenas eleifend erat at justo fringilla.</p>
+            <p class="milk_description-second">Curabitur lacinia enim at ex blandit, vel pellentesque odio elementum. Mauris rhoncus orci in
+                imperdiet placerat.
+                Vestibulum euismod nisl suscipit ligula volutpat, a feugiat urna maximus. Cras massa nibh, tincidunt.</p>
+            <p class="milk_description-third pic">Aliquam erat volutpat. Aenean accumsan.</p>
+                <button class="milk_btn" type="button">Read more</button>
+        </div>
+            </div>
+            <div class="cards">
+            <div class="card">
+                <ul>
+            <li><img class="card_img" src="./images/mobile-about/icon1.png" srcset="./images/mobile-about/icon1@2x.png 2x" alt="glass with milk" width="33px" height="50px">
```

Comment:
W Zadaniu Technicznym: "Ikony korzyści w układzie są renderowane w formacie png. W układzie muszą być zaimplementowane z pseudo-elementami."
Jako, że są pseudoelementami to trzeba je dać potem w css używając np. ::before, tutaj masz link, gdzie w odpowiedziach pokazują podobny problem: https://stackoverflow.com/questions/6668577/using-before-css-pseudo-element-to-add-image-to-modal